megamek/src/megamek/server/Server.java

```diff
@@ -26669,6 +26720,18 @@ else if (secondaryEffects) {
                     }
                     r.choose(true);
                     reports.add(r);
+                    // Lets auto-eject if we can!
+                    if (aero.isFighter()) {
+                        IAero ftr = (IAero) aero;
```

Comment:
`aero` is already of type Aero. Casting it to an interface it implements serves no purpose.

Reply:
Sure enough. Good catch.